pkg/apis/troubleshoot/v1beta2/remote_collector_shared.go

```diff
@@ -155,6 +155,10 @@ type RemoteServices struct {
 	RemoteCollectorMeta `json:",inline" yaml:",inline"`
 }
 
+type RemoteSysctl struct {
+	RemoteCollectorMeta `json:",inline" yaml:",inline"`
+}
+
```

Comment:
We can get rid of the changes in this file. This code path is going to be deprecated hopefully soon

Reply:
@diamonwiggins should I remove it right now before merge? Or do you mean that soon we'll be able to remove it?

Reply:
+1